src/WebJobs.Script/OutOfProc/Http/HttpInvokerWorkerContext.cs

```diff
@@ -6,11 +6,13 @@
 namespace Microsoft.Azure.WebJobs.Script.OutOfProc.Http
 {
     // Arguments to start a worker process
-    internal class HttpWorkerContext : WorkerContext
+    internal class HttpInvokerWorkerContext : WorkerContext
     {
+        public int HttpInvokerPort { get; set; }
+
         public override string GetFormatedArguments()
         {
-            return $" --workerId {WorkerId} --requestId {RequestId}";
+            return $" --httpInvokerPort {HttpInvokerPort}";
```

Comment:
Can we just make this param "--port" to make it simpler for the user's program? Nobody using this feature will understand what a http invoker is.

Reply:
Also are we setting the port as an environment variable on that process? We should, so that the user program can just refer to that environment variable instead of having to read the command line arguments.

Reply:
> Can we just make this param "--port" to make it simpler for the user's program? Nobody using this feature will understand what a http invoker is.

--port felt too generic. If that makes more sense, will update!

> Also are we setting the port as an environment variable on that process? We should, so that the user program can just refer to that environment variable instead of having to read the command line arguments.

restricting to environment variable will make it harder to enable "multiple http invoker processes". Should we have both? 
- Set environment variable for single invoker
- Detect via host.json setting if opting into multiple workers. If yes, pass command line args?